### PR TITLE
[MTDSA 12037] RAML for Create a Historic FHL UK Property Income & Expenses Period Summary

### PR DESCRIPTION
**Pull request by sean-flynn-HMRC**
Please note finalised endpoint description is pending.

### CODE REVIEW

File: resources/public/api/conf/2.0/examples/uk_property_historic_fhl_period_summary_create_and_amend/consolidated_request.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (mattstephens-hmrc) [inline resources/public/api/conf/2.0/examples/uk_property_historic_fhl_period_summary_create_and_amend/consolidated_request.json:1]: can the file just be auto-formatted in Intellij to remove the empty top line?
(Also technically ought to have a newline at the end of the last line)

File: resources/public/api/conf/2.0/schemas/uk_property_historic_fhl_period_summary_create_and_amend/request.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
+{
+  "title": "request",
+  "description": "Hi there. I'm waiting for the endpoint description to be finalised. Please continue with your review.",
```
Comment (mattstephens-hmrc) [inline resources/public/api/conf/2.0/schemas/uk_property_historic_fhl_period_summary_create_and_amend/request.json:3]: Is it possible to use the current non-finalised description in the meantime? (Albeit without the "use the XXX endpoint" sentence).  Just seen "temporary" text like this get merged into projects too often....

**Review by mattstephens-hmrc — APPROVED** [review]

File: resources/public/api/conf/2.0/examples/uk_property_historic_fhl_period_summary_create_and_amend/consolidated_request.json (hunk 1 above)
Comment (sean-flynn-HMRC) [reply resources/public/api/conf/2.0/examples/uk_property_historic_fhl_period_summary_create_and_amend/consolidated_request.json]: Formatted

File: resources/public/api/conf/2.0/schemas/uk_property_historic_fhl_period_summary_create_and_amend/request.json (hunk 2 above)
Comment (sean-flynn-HMRC) [reply resources/public/api/conf/2.0/schemas/uk_property_historic_fhl_period_summary_create_and_amend/request.json]: Updated text

**Pull request merged**